subsys/net/ip/connection.c

```diff
@@ -470,14 +457,14 @@ static inline void send_icmp_error(struct net_pkt *pkt)
 	}
 }
 
-static bool is_valid_packet(struct net_pkt *pkt,
-			    union net_ip_header *ip_hdr,
-			    u16_t src_port,
-			    u16_t dst_port)
+static bool conn_valid_end_points(struct net_pkt *pkt,
```

Comment:
I do not think we need the "conn_* prefix here. Better would be "are_valid_end_points()" or if you really want "conn_are_valid_end_points()"

Reply:
let's go for conn_are_end_points_valid()